### ISSUE TITLE
Notice button not working sometimes

### ISSUE THREAD

**Issue by adioni** [typemill/typemill#267]
When Selecting "Notice" from the edit toolbar and then clicking on the ' ! ' button to change the notice type, sometimesd nothing happens. Looking at the console in dev tools the following error messages are logged:

```
vue.min.js?20210613:6 TypeError: Cannot read property 'length' of undefined
    at a.getNoticePrefix (vue-blox.js?20210613:879)
    at a.noticedown (vue-blox.js?20210613:867)
    at click (eval at Ya (vue.min.js?20210613:6), <anonymous>:3:462)
    at He (vue.min.js?20210613:6)
    at HTMLButtonElement.n (vue.min.js?20210613:6)
    at HTMLButtonElement.Yr.o._wrapper (vue.min.js?20210613:6)
```
This is happening sporadically; 99.9% the time the notice button does/has been work fine. Maybe just a local caching issue I'm overlooking or something with the develop branch install I've been testing, but thought I should report it just in case.

**Comment by trendschau** [comment]
thank you, yes, there is at least one bug in the notice component when it is initially opened and still empty, then the up/down-button does not work correctly. I have to fix it for the next release.

**Comment by trendschau** [comment]
this is fixed in develop branch now and will be shipped with upcomming version 1.4.7

**Comment by trendschau** [comment]
fixed with version 1.4.7 (just published)
